compiler/src/dotty/tools/dotc/sbt/ExtractDependencies.scala

```diff
@@ -424,13 +424,30 @@ class DependencyRecorder {
       classDependencies.foreach(recordClassDependency(cb, _))
     clear()
 
+  private val _siblingClassfiles = new mutable.HashMap[PlainFile, Path]
+
+  /**Constructs a sibling class to the `jpath`.
+   * Does not validate if it exists as a real file.
+   * The way this works is that by the end of compilation analysis,
+   * there should be a corresponding NonLocalClass sent to zinc with the same class file name.
+   *
+   * FIXME: we still need a way to resolve the correct classfile when we split tasty and classes between
+   * different outputs (e.g. stdlib-bootstrapped).
+   */
+  private def siblingClass(pf: PlainFile): Path =
```

Comment:
Call this cachedSiblingClass or something else that makes it clear it's cached

Reply:
done